### PR TITLE
feat: support parameters for githubPrCheckApproved

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

* Add support to pass parameters to githubPrCheckApproved.

## Why is it important?

This allows calling the step from everywhere.

### CODE REVIEW

File: vars/githubPrCheckApproved.groovy [hunk 1]
```diff
@@ -24,14 +24,18 @@
   githubPrCheckApproved()
 */
 def call(Map params = [:]){
-  if(!isPR()){
+  def changeId =  params.get('changeId', env.CHANGE_ID)
+  def org = params.get('org', env.ORG_NAME)
+  def repo = params.get('repo', env.REPO_NAME)
+  def token = params.get('token', getGithubToken())
```
Comment (v1v) [inline vars/githubPrCheckApproved.groovy:30]: NIT: can you rename `params` for `args`?

**Review by v1v — APPROVED** [review]

File: vars/README.md [hunk 2]
```diff
@@ -868,6 +868,15 @@ githubPrCheckApproved()
 
 NOTE: `REPO_NAME` env variable is required, so gitHubEnv step is the one in charge
 
+```
+githubPrCheckApproved(org: 'elastic', repo: 'apm-pipeline-library', changeId: 1000, token: "env.GITHUB_TOKEN")
+```
+
+* *org:* GitHub organization/owner of the repository (by default ORG_NAME).
+* *repo:* GitHub repository name (by default REPO_NAME).
+* *changeId:* Pull request Id number (by default CHANGE_ID).
```
Comment (cachedout) [inline vars/README.md:877]:
```suggestion
* *changeId:* Pull request ID number (by default CHANGE_ID).
```

File: vars/githubPrCheckApproved.txt [hunk 3]
```diff
@@ -8,3 +8,12 @@ githubPrCheckApproved()
...
 
 NOTE: `REPO_NAME` env variable is required, so gitHubEnv step is the one in charge
+
+```
+githubPrCheckApproved(org: 'elastic', repo: 'apm-pipeline-library', changeId: 1000, token: "env.GITHUB_TOKEN")
+```
+
+* *org:* GitHub organization/owner of the repository (by default ORG_NAME).
+* *repo:* GitHub repository name (by default REPO_NAME).
+* *changeId:* Pull request Id number (by default CHANGE_ID).
```
Comment (cachedout) [inline vars/githubPrCheckApproved.txt:18]:
```suggestion
* *changeId:* Pull request ID number (by default CHANGE_ID).
```

**Review by cachedout — APPROVED** [review]

**Pull request merged**